Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -73,6 +73,17 @@ static void wipeAlphaChannelFromPixels(std::span<uint8_t> pixels)
 }
 #endif
 
+static inline const Vector<const void*> asPointers(std::span<const GCGLsizei> offsets)
+{
+    // Must cast offsets from int to void* before passing down to ANGLE.
+
+    Vector<const void*> pointers;
+    pointers.reserveInitialCapacity(offsets.size());
```

Comment:
Ideally, this would be using `WTF::map()`, which is potentially more efficient because it avoids the capacity checks in `append()`.

Reply:
Less code too. I can do that.